api/api/urls.py

```diff
@@ -10,6 +10,7 @@
     path('api-auth/', include('rest_framework.urls', namespace='rest_framework')),
     path('auth/', include('api.auth.urls')),
     path('ping/', ping_handler, name='ping'),
+    path('clubs/', include('api.clubs.urls'))
```

Comment:
Let's keep these paths in alphabetical order. It would be easy for others if they're trying to find a path.
```suggestion
    path('clubs/', include('api.clubs.urls')),
    path('ping/', ping_handler, name='ping'),
```